python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -272,19 +282,23 @@ def fitBackground(self, maskedImage, nx=0, ny=0, algorithm=None):
         """!Estimate the background of a masked image
 
         @param[in] maskedImage  masked image whose background is to be computed
-        @param[in] nx  number of x bands; if 0 compute from width and config.binSize
-        @param[in] ny  number of y bands; if 0 compute from height and config.binSize
+        @param[in] nx  number of x bands; if 0 compute from width and config.binSizeX
+        @param[in] ny  number of y bands; if 0 compute from height and config.binSizeY
         @param[in] algorithm  name of interpolation algorithm; if None use self.config.algorithm
 
         @return fit background as an lsst.afw.math.Background
 
         @throw RuntimeError if lsst.afw.math.makeBackground returns None,
             which is apparently one way it indicates failure
         """
+
+        binSizeX = self.config.binSizeX if self.config.binSizeX else self.config.binSize
+        binSizeY = self.config.binSizeY if self.config.binSizeY else self.config.binSize
```

Comment:
Add explicit `== 0`?  Makes it clearer and corresponds more exactly with your config docs.